features/string.yml

```diff
@@ -0,0 +1,7 @@
+name: string
```

Comment:
I think we'll likely need a disambiguation on this name and id. Probably on a few others as well.